scripts/events_module/handle_short_events.py

```diff
@@ -317,14 +317,14 @@ def handle_accessories(self):
             if acc not in ["WILD", "PLANT", "COLLAR"]:
                 acc_list.append(acc)
 
-        if "NOTAIL" in self.main_cat.pelt.scars or "HALFTAIL" in self.main_cat.pelt.scars:
-            for acc in Pelt.tail_accessories:
-                try:
-                    acc_list.remove(acc)
-                except ValueError:
-                    print(f'attempted to remove {acc} from possible acc list, but it was not in the list!')
+        if getattr(getattr(self.main_cat, "pelt", None), "scars", None):
```

Comment:
May want to investigate `hasattr` instead of `getattr` here, at least for this outer one. Makes it a bit clearer what you're trying to do, and I can't find any documentation on the truthiness of `getattr`

Reply:
Thanks.  `hasattr` is definitely clearer and will always be True or False.  Whereas `getattr` was only able to be False because of my setting the default to None.  I may remove the inner clause for now, as it works in it's current form. If I can mock the cat out of the test, I'll check for it then.